src/jquery.smoothState.js

```diff
@@ -505,7 +505,30 @@
             window.location = url;
           }
         },
+        /**Reload the page */
+        reload = function () {
+          // Apply rate limiting.
+          if (!isRateLimited()) {
 
+            // Set the delay timeout until the next event is allowed.
+            setRateLimitRepeatTime();
+
+            // clear cache
+            cache = {};
```

Comment:
What if someone's cache has 4 or 5 entries, will this wipe the entire thing? Possible to search for the current location and remove only the one entry?